### ISSUE TITLE
Messages should disappear when pressing ENTER

### ISSUE THREAD

**Issue by farooqkz** [minetest/minetest#8930]
## Problem
I'm talking about these sort of messages:
![screen03 png](https://user-images.githubusercontent.com/15038218/64882030-78eda700-d671-11e9-9c50-147e7b386ec9.jpg)

They don't disappear when user presses ENTER.

## Solution
These sort of messages should disappear when user presses ENTER.

**Comment by paramat** [comment]
I disagree they 'should' disappear on pressing enter, that's not particualrly obvious or intuitive, however i don't mind if they do.
Disappearing on 'escape' would be more obvious and intuitive. 'Enter' is more for submitting entered information.

**Comment by DS-Minetest** [comment]
This is probably very easy to do. The button has to be focused.
However, it is possible that the enter key will immediately trigger the focused element of the next screen. (This happens for example if you create a new world and use enter.)

**Comment by farooqkz** [comment]
> I disagree they 'should' disappear on pressing enter, that's not particualrly obvious or intuitive, however i don't mind if they do.
> Disappearing on 'escape' would be more obvious and intuitive. 'Enter' is more for submitting entered information.

I wanted to open another issue for that. Ask me if I should. In this situation pressing Escape key MUST NOT close Minetest.

**Comment by paramat** [comment]
Hang on, i'm somewhat wrong. General interface behaviour does tend to activate a button if the button is highlighted, which it often is if it is the only button. Sorry.

**Comment by ClobberXD** [comment]
I was going to make this (an) issue sooner or later, thanks. I feel exactly the same way as the OP does, and I'd love it if I could press Enter to dismiss it.
